lib/fast_jsonapi/serialization_core.rb

```diff
@@ -51,37 +51,38 @@ def attributes_hash(record, fieldset = nil, params = {})
         end
       end
 
-      def relationships_hash(record, relationships = nil, fieldset = nil, params = {})
+      def relationships_hash(record, relationships = nil, fieldset = nil, includes_list = nil, params = {})
         relationships = relationships_to_serialize if relationships.nil?
         relationships = relationships.slice(*fieldset) if fieldset.present?
         relationships = {} if fieldset == []
 
-        relationships.each_with_object({}) do |(_k, relationship), hash|
-          relationship.serialize(record, params, hash)
+        relationships.each_with_object({}) do |(key, relationship), hash|
+          included = includes_list.present? && includes_list.include?(key)
+          relationship.serialize(record, included, params, hash)
         end
       end
 
       def meta_hash(record, params = {})
         meta_to_serialize.call(record, params)
       end
 
-      def record_hash(record, fieldset, params = {})
+      def record_hash(record, fieldset, includes_list, params = {})
```

Comment:
If includes_list doesn't have a default value here, it could be a breaking change.